roles/migrationcontroller/templates/velero_supporting.yml.j2

```diff
@@ -15,6 +15,7 @@ spec:
     singular: backup
   preserveUnknownFields: false
   scope: Namespaced
+{% if lookup('k8s', cluster_info='version').kubernetes.minor|replace('+', '')|int < 11 %}
```

Comment:
Should we be comparing the major version too? This will break with kubernetes 2.0